modules/pipeline/spec/pipeline_extra.go

```diff
@@ -70,8 +70,9 @@ type PipelineExtraInfo struct {
 	// Deprecated
 	ConfigManageNamespaceOfSecretsDefault string `json:"configManageNamespaceOfSecretsDefault,omitempty"`
 	// Deprecated
-	ConfigManageNamespaceOfSecrets string   `json:"configManageNamespaceOfSecrets,omitempty"`
-	ConfigManageNamespaces         []string `json:"configManageNamespaces,omitempty"`
+	ConfigManageNamespaceOfSecrets string            `json:"configManageNamespaceOfSecrets,omitempty"`
+	ConfigManageNamespaces         []string          `json:"configManageNamespaces,omitempty"`
+	Secrets                        map[string]string `json:"secrets,omitempty"`
```

Comment:
use `requestLevelSecrets` will be better to distinguish with secrets from configNamespaces.